app/javascript/dashboard/routes/dashboard/settings/canned/EditCanned.vue

```diff
@@ -133,11 +133,10 @@ export default {
             this.onClose();
           }, 10);
         })
-        .catch(() => {
+        .catch(error => {
           this.editCanned.showLoading = false;
-          this.editCanned.message = this.$t(
-            'CANNED_MGMT.EDIT.API.ERROR_MESSAGE'
-          );
+          this.editCanned.message =
+            error?.message || this.$t('CANNED_MGMT.EDIT.API.ERROR_MESSAGE');
           this.showAlert();
```

Comment:
Please remove the component method here and use the mixin.

Reply:
Thanks for pointing out the mistake, I have moved it inside the if block. It was done as we were having multiple email copies within the same mail and the issue's copy changes are different from the existing one. This would make sure the exiting email copies does not change.

Reply:
I'll make the change